src/app/shared/palette/utils/date.pipe.ts

```diff
@@ -12,13 +12,19 @@ export class DatePipe implements PipeTransform {
     ['Jan'], ['Feb'], ['Mar'], ['Apr'], ['May'], ['Jun'], ['Jul'], ['Aug'], ['Sep'], ['Oct'], ['Nov'], ['Dec'],
   ];
 
-  transform(value: string, format: string): string {
+  transform(value: string, zone: string, format: string): string {
```

Comment:
Can we have one more function

transform(value: string, format: string): string {
    return transform(value, format, null)
}

Reply:
As discussed, the method overloading is not supporting in angular.

Reply:
thx